src/Tribe/Views/V2/View.php

```diff
@@ -820,7 +839,14 @@ public function prev_url( $canonical = false, array $passthru_vars = [] ) {
 
 		if ( ! empty( $passthru_vars ) && ! empty( $url ) ) {
 			// Re-apply the pass-thru query arguments.
-			$url = add_query_arg( $passthru_vars, $url );
+			$query_args = array_merge( $query_args, $passthru_vars );
+		}
+
+		$query_args = $this->filter_query_args( $query_args, $url );
+		$query_args = array_filter( $query_args );
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: ec4a31c8a2f5c2923e12b0365c2b2fd2</sub>